install/connections.csv

```diff
@@ -835,6 +835,14 @@ RU001,01,Ruapehu Crater Lake,,,1993-12-11T00:00:00Z,9999-01-01T00:00:00Z
 RU001,02,Ruapehu Crater Lake,,,2009-04-15T02:00:00Z,2011-06-20T23:00:00Z
 RU001,03,Ruapehu Crater Lake,,,2016-03-04T03:15:00Z,9999-01-01T00:00:00Z
 RU001,04,Ruapehu Crater Lake,,,2019-05-21T04:30:00Z,9999-01-01T00:00:00Z
+RU001,05,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,06,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,07,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,08,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,09,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,10,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,11,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
+RU001,12,Ruapehu Crater Lake 2,,,2025-05-13T22:20:00Z,9999-01-01T00:00:00Z
```

Comment:
(see main comment and the one in `install/dataloggers.csv`) 
change each line from
`xxxx,Ruapehu Crater Lake 2,,,2025-.....xxxx`
to
`xxxx,Ruapehu Crater Lake,Envirosensor,,2025-.....xxxx`

Reply:
included in updated version, resolved